docs/cdp/cdp_guide.md

```diff
@@ -0,0 +1,134 @@
+
+- **AND** - use when you want to intersect two or more values for a particular segment. All set conditions must be met.
+- **OR** - use when you want to broaden results, one of the conditions must be fulfilled.
+
+Segments available in the Elements sections are reusable. It means you can use the same segment in different segment groups.
+
+![Segment groups](img/segment_groups.png)
+
+## Capabilities
+
+### Data export
+
+Configuration in [[= product_name_cdp =]] allows you to automate the process of exporting Content, Users, and Products.
+Under the schedule setting you can find separate sections for exporting User, Content, and Product. Structure of each section is exactly the same and includes interval and options elements:
+
+- Interval - sets the frequency of the command invoke, for example, '*/30 * * * *' means "every 30 minutes", '0 */12 * * *' means "every 12 hours".
+
+- Options - allows you to add arguments that have to be passed to the export command.
+
+This configuration allows you to provide multiple export workflows with parameters. It's important, because all the types of content/product must have their own parameters on the CDP side, where each has a different Stream ID key and different required values configured per data source.
+
+To check all the process of data export, see [data export](https://doc.ibexa.co/en/latest/cdp/cdp_activation/cdp_data_export/).
+
+### Data customization
+
+​You can customize Content and Product data exported to [[= product_name_cdp =]] and control what Field Type information you want to export.
+With [[= product_name_cdp =]] you can export Field Types and Field Type values. They are exported with metadata, for example, ID, Field Definition name, type, value.
+
+For more information, see [data customization](https://doc.ibexa.co/en/latest/cdp/cdp_data_customization/#data-customization) documentation in Developer Documentation.
+
+### Client-side Tracking
+
+The final step is setting up a tracking script.
+It requires a head tracking script on your website, a main script after the head script, and cookie consent.
+For more information about setting up a tracking script, see [a tutorial in Raptor documentation](https://support.raptorsmartadvisor.com/hc/en-us/articles/9563346335004-Client-Side-Tracking).
+
+### Audience Builder
+
+In the Audience Builder, you can create audiendes - groups of users that meet the assumed conditions. You can choose specific conditions: `did`, `did not`, or `have`.
+The conditions `did` and `did not` allow you to use events like buy, visit or add to a cart from online tracking. The  `have` conditions are tied to personal characteristics and can be used to track the sum of all buys or top-visited categories. 
+
+You can also connect created audiences to the activations.
+
+![Audience Builder](img/audience_builder.png)
+
+## Limitations
+
+[[= product_name_cdp =]] doesn't support column mapping, which allows you to match records on JSON data directly.
```

Comment:
```suggestion
When exporting Field Type values, [[= product_name_cdp =]] doesn't support column mapping, which allows you to match records on JSON data directly.
```
If "column mapping allows you to match...", I guess the latter part of the sentence should say ", which would allow you to match records on JSON data directly."
Otherwise it sounds as if the lack of support allows this.